src/main/java/org/ossgang/commons/mapbackeds/Mapbackeds.java

```diff
@@ -157,23 +245,40 @@ public M build() {
     }
 
     public static Set<Method> fieldMethods(Class<?> intfc) {
+        return collectIntfcMethods(intfc, m -> !m.isDefault());
+    }
+
+    public static Optional<Method> toStringMethod(Class<?> intfc) {
```

Comment:
private?

Reply:
It is used from the MapbackedObject class .. indeed, there is a circular dependency between the two ... Not sure if this worth resolving at the moment ... the Mapbackeds class contains both, utility methods and factory methods ....

Reply:
I see ... one quick thing could be to make the utility method package private - so they won't be seen by outside users. Of perhaps move them to a package-private utility class ...

My concern is just that if we expose a utility methods once, we will never be able to change, rename or even get rid of them anymore without breaking code in other places ;-)

Reply:
Agreed. I will recheck...